cypress/integration/missing.assets.feedback.spec.js

```diff
@@ -0,0 +1,77 @@
+context("The Readalong Component", () => {
```

Comment:
Please provide a distinctive title for the `context()` call. This makes groups of test cases easy to identify in the Cypress panel:

```suggestion
context("Missing Readalong assets", () => {
```

Otherwise it looks like this:

![Screen Shot 2021-07-28 at 9 04 33 AM](https://user-images.githubusercontent.com/2294397/127347303-c2403869-e078-4616-949f-9d00b9400f44.png)